src/lib/config/definition.ts

```diff
@@ -3,6 +3,10 @@ const defaultWindowRules = `[
         "class": "(org\\\\.kde\\\\.)?plasmashell",
         "tile": false
     },
+    {
+        "class": "org.kde.polkit-kde-authentication-agent-1",
```

Comment:
```suggestion
        "class": "(org\\\\.kde\\\\.)?polkit-kde-authentication-agent-1",
```

This makes it match on X11 as well.